Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -214,4 +276,26 @@ img.disabled,
 {
   opacity: 0.5;
   filter: grayscale(100%);
-}
\ No newline at end of file
+}
+
+.blink .remotion-themed span[role=checkbox]:focus > img,
+.blink .remotion-themed span[role=radio]:focus > img,
+.blink .remotion-themed input[type=checkbox]:focus + span:before,
+.blink .remotion-themed input[type=checkbox]:focus + label:before,
+.blink .remotion-themed input[type=radio]:focus + span:before,
+.blink .remotion-themed input[type=radio]:focus + label:before
+{
+  box-shadow: var(--chromium-box-shadow);
+  border-radius: var(--chromium-border-radius);
+}
+
+.mozilla .remotion-themed span[role=checkbox]:focus > img,
+.mozilla .remotion-themed span[role=radio]:focus > img,
+.mozilla .remotion-themed input[type=checkbox]:focus + span:before,
+.mozilla .remotion-themed input[type=checkbox]:focus + label:before,
+.mozilla .remotion-themed input[type=radio]:focus + span:before,
+.mozilla .remotion-themed input[type=radio]:focus + label:before
+{
+  box-shadow: var(--mozilla-box-shadow);
+  border-radius: var(--mozilla-border-radius);
+}
```

Comment:
This code is dead once you rebase on top of the NovaViso branch since the focus-task is already implemented now.